src/seedu/addressbook/ui/TextUi.java

```diff
@@ -13,31 +13,19 @@
 import java.util.Optional;
 import java.util.Scanner;
 
+import static seedu.addressbook.ui.Formatter.LINE_PREFIX;
+import static seedu.addressbook.ui.Formatter.DIVIDER;
+import static seedu.addressbook.ui.Formatter.LS;
```

Comment:
Try to make the TextUi class totally unaware of these formatting-related constants? Note that it may not be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.